README.md

```diff
@@ -130,6 +135,52 @@ Variant3 bar: I was created in a function!
 Variant3 baz: Some("1")
 ```
 
+## Accessing enum fields in `assoc` attribute
+
+It is possible to access an enum variant field value in assoc attribute
+prefixing its name with an underscore. For tuples the names is composed of
+underscore prefix and field index.
+
+```rust,ignore
+use thiserror::Error;
+
+#[derive(Error, Debug, Assoc, Clone)]
+#[func(pub const fn status(&self) -> u16)]
+pub enum ServiceError {
+    #[error("failed to start or finish a transaction")]
+    #[assoc(status = 500)]
+    TransactionError { source: sea_orm::DbErr },
```

Comment:
I think that an example that does not use external dependencies would be a bit easier

Reply:
Agreed, simple is better.